tox.ini

```diff
@@ -1,7 +1,7 @@
 [tox]
 envlist = tests
 skipsdist = true
-minversion = 3.8.0
+minversion = 3.16.1
```

Comment:
After people `git pull` the next time they run a `tox` command tox will automatically update itself to 3.16.1 or newer. I did this because I've added some tox options below that are new in recent versions of tox